packages/allure-jest/test/default-runner/simple.test.ts

```diff
@@ -1,3 +1,5 @@
+// eslint-disable-next-line no-undef
```

Comment:
Should be fixed with `/*eslint-env jest*/`?

Reply:
done